src/rust/devcontainer-feature.json

```
@@ -1,6 +1,6 @@
 {
     "id": "rust",
-    "version": "1.1.3",
+    "version": "1.1.4",
```

```suggestion
    "version": "1.2.0",
```

Can we bump minor version as we are adding a new Feature option?